src/odemis/dataio/csv.py

```diff
@@ -62,13 +63,26 @@ def export(filename, data):
             csv_writer = csv.writer(fd)
             csv_writer.writerow(headers)
             csv_writer.writerows(spectrum_tuples)
-    elif data.ndim == 2 and all(s >= 2 for s in data.shape):
+    elif data.ndim == 2 and all(s >= 2 for s in data.shape) and (data.metadata.get(model.MD_DESCRIPTION, None) == "Angle-resolved"):
```

Comment:
We should never rely on the MD_DESCRIPTION to detect the type of data.
Use the fact that the spectrum has MD_WL_LIST present.
Probably we should also export the distance from the origin (for spectrum line) and Theta/Phi angles as metadata. Up to you to do it now or we'll do it another time.